Workbooks/Azure Blockchain/Workbench Troubleshooting/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+    "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
+    "name":"Workbench Troubleshooting",
+    "author": "Azure Blockchain",
+    "order": 100,
```

Comment:
You probably don't need this anymore. @ericc1103, can you confirm?

Reply:
Right, we don't need it anymore since we already specify it within galleries.

Reply:
sounds good - I'll remove it